src/shared/components/home/login.tsx

```diff
@@ -155,8 +155,14 @@ export class Login extends Component<any, State> {
             i.setState({ showTotp: true });
             toast(I18NextService.i18n.t("enter_two_factor_code"), "info");
           }
-          if (loginRes.msg === "incorrect_login") {
-            toast(I18NextService.i18n.t("incorrect_login"), "danger");
+          if (
```

Comment:
You could probably just remove these extra checks, and do an `else { toast(...`

Reply:
Yea, that sounds better, in that case even when the localization for the key doesn't exist then it will just display the key itself I think. I've pushed the changes.